src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocpackage/nojavadoc/annotation/blockcomment/package-info.java

```diff
@@ -0,0 +1,8 @@
+/*
+*  Some comment
```

Comment:
Please add `violation` marker here. This will help to separate the positive inputs from the negative ones for those who will work with this in the future.

Reply:
I put violation comment on package def line (because violation is reported on this token)

Also added violation comment for other tests for this check